src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -476,6 +476,14 @@ export class NotebookModel extends Disposable implements INotebookModel {
 		}
 	}
 
+	public createNotebookTelemetryEvent(action: TelemetryKeys.TelemetryAction | TelemetryKeys.NbTelemetryAction, additionalProperties?: ITelemetryEventProperties): void {
+		let properties: ITelemetryEventProperties = additionalProperties;
+		properties['azdata_notebook_guid'] = this.getMetaValue('azdata_notebook_guid');
```

Comment:
properties might be undefined here if the user didn't pass an object in. Default to empty object for that case on 480